### ISSUE TITLE
http should redirect to https for `dev.rotisserie.tv`

### ISSUE THREAD

**Issue by MHBauer** [IBM/rotisserie#123]
`http://dev.rotisserie.tv` is the default nginx landing page.

the `Thanks` slide at https://blog.eggshell.me/rotisserie-talk/#39
links to `dev.rotisserie.tv` which by default is interpreted as http when you copy and paste it in.
`http://dev.rotisserie.tv`

It should be a direct clicky link to `https://dev.rotisserie.tv`

Additionally `http://dev.rotisserie.tv` should redirect to `https://dev.rotisserie.tv`.

 - [x] fix slides with direct link to `https://dev.rotisserie.tv`
 - [x] make `http://dev.rotisserie.tv` redirect to `https://dev.rotisserie.tv`

**Comment by mpetason** [comment]
Dev is for staging. Right now we are using Cert-Manager + Kube-Lego in the prod cluster, where dev also lives, which causes the issues with redirects. Cert-Manager + the IBM ingress controller should be controlling all traffic for the domain, however Kube-Lego also sees the annotation and is setting up ingress resources for the domain. There is currently a bug with the Ingress Controller which keeps it from pulling records for the same host from two different ingress resources. 

Basically - As soon as PR #122 is merged and we re-deploy prod + dev + fortnite, we shouldn't run into this issue. The setup already assumes HTTP to HTTPS redirects.

**Comment by eggshell** [comment]
Went ahead and updated the last slide. Thanks @MHBauer

**Comment by MHBauer** [comment]
confirm slide update.

**Comment by mpetason** [comment]
http redirects to https after removing kube-lego and switching everything over to cert-manager.

**Comment by MHBauer** [comment]
confirm redirect.

thanks guys. good job!
